README.md

```diff
@@ -10,7 +10,7 @@
 
 <p align="center">
   <a
-    href="https://open.mp/wiki"
+    href="https://wiki.openmultiplayer.now.sh"
```

Comment:
This is the internal URL, the actual site is on https://open.mp/docs

the /wiki redirect is being fixed right now.